README.rst

```diff
@@ -36,64 +35,25 @@ Installation
 
     pip install web-poet
 
-Usage
-=====
-
-Check the following script that uses ``urllib.request`` to query data from
-`books.toscrape.com`_.
-
-.. code-block:: python
-
-    import urllib.request
-
-    from web_poet.pages import ItemWebPage
-    from web_poet.page_inputs import ResponseData
+It requires Python 3.6+.
 
+Overview
+========
 
-    class BookLinksPage(ItemWebPage):
-
-        @property
-        def links(self):
-            return self.css('.image_container a::attr(href)').getall()
-
-        def to_item(self) -> dict:
-            return {
-                'links': self.links,
-            }
-
-    response = urllib.request.urlopen('http://books.toscrape.com')
-    response_data = ResponseData(response.url, response.read().decode('utf-8'))
-    page = BookLinksPage(response_data)
+web-poet is a library which defines a standard on how to write and organize
+web data extraction code.
 
-    print(page.to_item())
+If web scraping code is written as web-poet Page Objects, it can be reused
+in different contexts. For example, such code can be developed in an
+IPython notebook, then tested in isolation, and then plugged
```

Comment:
It would be nice to add references to IPython notebook (Jupyter?) and aiohttp here.